net-wireless/nitroshare/nitroshare-0.4.0~git.recipe

```diff
@@ -60,15 +67,19 @@ defineDebugInfoPackage nitroshare$secondaryArchSuffix \
 
 BUILD()
 {
-	cmake -DCMAKE_INSTALL_PREFIX:PATH=$appsDir .
+	cmake -DCMAKE_BUILD_TYPE=Release \
+	-DINSTALL_INCLUDE_PATH=$includeDir
```

Comment:
I know it's disabled for 32bit (doesn't launch anyway there) :)
but maybe you could add just the same (in case it's fixed sometime)
```diff
+	-DINSTALL_BIN_PATH=$binDir
+	-DINSTALL_LIB_PATH=$libDir
```